app/mailers/previews/provider_mailer_preview.rb

```diff
@@ -1,7 +1,24 @@
 class ProviderMailerPreview < ActionMailer::Preview
   def account_created_email
-    provider_user = FactoryBot.build :provider_user
-
     ProviderMailer.account_created(provider_user)
   end
+
+  def application_submitted
+    ProviderMailer.application_submitted(provider_user, application_choice)
+  end
+
+  def application_rejected_by_default
+    ProviderMailer.application_rejected_by_default(provider_user, application_choice)
+  end
+
+private
+
+  def application_choice
+    course_option = FactoryBot.create(:course_option, course: FactoryBot.build(:course))
+    FactoryBot.create(:submitted_application_choice, course_option: course_option)
```

Comment:
Can we `build` here instead? Otherwise visiting the mailer test pages will generate records on QA.

Reply:
I used `create` because `build` did not allow to associate the `application choice` with the `course`. Any idea, what would be a better workaround?

Reply:
Not sure, let's figure that out later. Approved!